README.md

```diff
@@ -75,6 +75,10 @@ This repo contains teaching material and assignments for ReDI School in Munich J
 - Homework:
   - start creating online store page with some goods
 
+## [Lesson 8 - intro to objects](https://mrtim.github.io/js-munich-2020-spring/lessons/8_loops)
```

Comment:
should be `[Lesson 8 - Loops](https://mrtim.github.io/js-munich-2020-spring/lessons/8_loops)`